beta-prompts/beta_prompts/generation_system.py

```
@@ -0,0 +1,955 @@
+        )
+
+        return GeneratedPrompt(
+            prompt=prompt,
+            task_type=task_type,
+            domain=domain,
+            complexity=complexity,
+            elements_used=blueprint.elements,
+            quality_score=quality_score,
+            variables_used=variables_used,
+            generation_method="template_based",
+            timestamp=datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)